### PR TITLE
contains() and containsAll() methods added

### PR DESCRIPTION
**Pull request by Zulfiya5**

### CODE REVIEW

File: src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java [hunk 1]
```diff
@@ -13,4 +15,33 @@ public ArrayCollection(int capacity){
...
+    public boolean containsAll(Object[] values) {
+        for (Object value : values) {
+            if (!contains(value)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public void insert(Object item) {
```
Comment (com-sultanov-roman) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java]: It is better to rename insert() method as pushBack() as it is more intuitively clear

File: src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java [hunk 2]
```diff
@@ -13,4 +15,33 @@ public ArrayCollection(int capacity){
...
+        if (content.length == capacity) {
+            Object[] newContent = new Object[capacity * 2];
```
Comment (com-sultanov-roman) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java:43]: it is better to extract resize logic to another method

**Review by com-sultanov-roman — CHANGES_REQUESTED** [review]

**Review by com-sultanov-roman — APPROVED** [review]

File: src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java [hunk 3]
```diff
@@ -13,4 +15,35 @@ public ArrayCollection(int capacity){
         content=new Object[capacity];
     }
 
+    public boolean contains(Object value) {
```
Comment (xtreme-uz) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java:18]: What about null value, it is not null safe

File: src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java [hunk 4]
```diff
@@ -13,4 +15,35 @@ public ArrayCollection(int capacity){
...
+        }
+        return true;
+    }
+
+    public void pushBack(Object item) {
+        doubleContentLengthIfCapacityIsReached();
+        content[capacity++] = item;
+    }
+
+    public void doubleContentLengthIfCapacityIsReached() {
```
Comment (xtreme-uz) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java:41]: `capacity` should also be changed

Comment (xtreme-uz) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java:41]: You used `capacity` as `size` of array, check you logic

File: src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java [hunk 5]
```diff
@@ -13,4 +15,35 @@ public ArrayCollection(int capacity){
         content=new Object[capacity];
     }
 
+    public boolean contains(Object value) {
+        for (int i = 0; i < capacity; i++) {
```
Comment (MurodDevUzb) [inline src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java:19]: Почему до capacity проверяем

**Review by dostonbokhodirov — APPROVED** [review]

**Review by nurmukhammadsunatullaev — APPROVED** [review]

**Pull request merged**